Source/WebKit/glib/profiling.md.in

```diff
@@ -0,0 +1,106 @@
+Title: Profiling
+Slug: profiling
+
+# Profiling with sysprof
+
+## Prerequisites
+
+To work with sysprof one needs to install it first:
+
+- ubuntu:
+```
+apt install sysprof
+```
+- fedora
+```
+dnf install sysprof
+```
+
+## Profiling
+
+To start the profiling one must either:
+
+- start `sysprof-cli` to get a system-wide capture:
+```
+sysprof-cli -f
+```
+- or wrap a command with `sysprof-cli` to get a capture funneled to particular process tree created by the given command:
+```
+sysprof-cli -f -- sleep 3
```

Comment:
Let's put the MiniBrowser here, I think that would be clearer... it took me way too long thinking that `sleep 3` was there to record existing processes during 3 seconds until I realized that `sleep 3` was only an example command:

```suggestion
sysprof-cli -f -- MiniBrowser http://webkit.org
```